openBCISample.js

```diff
@@ -31,30 +31,39 @@ var sampleModule = {
   *                  calling OpenBCIConstans.channelSettingsArrayInit(). The most important rule here is that it is
   *                  Array of objects that have key-value pair {gain:NUMBER}
   * @param convertAuxToAccel (optional) {Boolean} - Do you want to convert to g's? (Defaults to true)
-  * @returns {Promise} - A sample object
```

Comment:
On line 603 the syntax `@returns {object}` is used, perhaps that should be filled in here too